### PR TITLE
Update create-a-topic.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a new line item to first click on the new topic page from the manage topics page before creating it.

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/4146

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/knowledge/create-a-topic.md [hunk 1]
```diff
@@ -49,15 +49,17 @@ You can create a new topic from two locations:
 
 ### To create a new topic:
 
-1. In the **Name this topic** section, type the name of the new topic.
+1. Select the option to create a new Topic Page from the ribbon on the Manage Topics page.
```
Comment (JohanFreelancer9) [inline microsoft-365/knowledge/create-a-topic.md:52]:
```suggestion
1. Select the option to create a new topic page from the ribbon on the **Manage topics** page.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Pull request merged**